packages/api/src/external/carequality/patient.ts

```diff
@@ -72,7 +136,7 @@ export async function handleDiscoverResponse(
     // WE SHOULD EXPECT THIS TO HAPPEN SO SHOULD WE ADD SENTRY MESSAGE?
     // DONT WANT TO BOMBARD ALERTS CHANNEL
     console.log(
```

Comment:
Forgot to mention this https://github.com/metriport/metriport/pull/1302#discussion_r1424284557

Reply:
uh oh..